test/unit/router/compliance.spec.ts

```
@@ -6,14 +6,14 @@ import {
     defineCoreHandler,
 } from '../../../src';
 import type { IRouter } from '../../../src';
```

ESLint

[error] '../../../src' imported multiple times.

(import/no-duplicates)